systems/subscriber/sim-manager/pkg/server/sim_manager_test.go

```diff
@@ -314,17 +329,26 @@ func TestSimManagerServer_AllocateSim(t *testing.T) {
 			SubscriberID: subscriberID,
 			NetworkID:    networkID,
 			OrgID:        orgID,
-			Type:         sims.SimType(simPoolResp.Sim.SimType),
+			Type:         1,
```

Comment:
Using the enum value (instead of the number value) will make it easier to update/debug in the future.

Reply:
addressed